.github/workflows/java-ci.yml

```diff
@@ -87,6 +87,11 @@ jobs:
       with:
         distribution: zulu
         java-version: 8
+    # Setup Gradle properties, heap requirements are for `:iceberg-spark:iceberg-spark-3.3_2.12:checkstyleMain`.
```

Comment:
Instead of updating the https://github.com/apache/iceberg/blob/master/gradle.properties, I have modified here because changing there will apply for all the tasks and other tasks doesn't have this requirement.